godotproject/Level0.tscn

```diff
@@ -24,7 +23,7 @@ Spike = ExtResource( 11 )
 Star = ExtResource( 9 )
 
 [node name="Player" parent="." instance=ExtResource( 1 )]
-position = Vector2( 110.163, 870.544 )
+position = Vector2( 118.904, 898.885 )
```

Comment:
The player's initial position was changed so that they wouldn't appear to be floating mid-air when the user loads the level.